motoko/ios-notifications/dapp-demo/internet_identity.did

```diff
@@ -0,0 +1,205 @@
+type UserNumber = nat64;
```

Comment:
This file and the `wasm` should not be checked in, right?

Reply:
true, will add to `.gitignore`